### DISCUSSION TITLE
Newbie to Github here but I think I have a problem.

### DISCUSSION THREAD

**Discussion by Bladevane** [community/community#30198] [New to GitHub] [answered]
I downloaded Prusa Slicer 2.5.0-rc1 for Windows and loaded up an STL which was 220 mm x 220 mm. When I tried to slice it the program closed. I reopened it and downloaded my STL again but set the scaling to 95% and slicing worked OK. However when set to 96% it closed again.

**Comment by airtower-luna** [comment, marked as answer]
It looks like you're looking for help with a software hosted on GitHub, maybe https://github.com/prusa3d/PrusaSlicer? In that case you should open an issue *in that repository* so the developers see it. :slightly_smiling_face:

**Comment by queenofcorgis** [comment]
Like @airtower-luna said, if you're asking how to use the piece of software that you found in a repository on GitHub, the best way to contact the maintainers of that software is to:

- Check the README for instructions on how to operate the software and pointers to documentation or troubleshooting info.
- Check the [SUPPORT file,](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-support-resources-to-your-project) if one exists, for instructions on how to best contact the maintainers for support.
- Check the [CONTRIBUTING guide](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/setting-guidelines-for-repository-contributors), if one exists. Sometimes if there isn't a SUPPORT file, the CONTRIBUTING guide will give instructions on how to contact the maintainers for support.

All of these documents can be found, if they exist, in the repository where you found the software itself.

**Reply by gayathriece0612** [reply]
Not come invitation email

**Comment by 1973darrell** [comment]
2.7.2 prusa slicer is corrupt on my windows 10hp
![20240302_134418](https://github.com/community/community/assets/16892813/c6fbfd11-b2f8-4b05-9d57-0d5250f90d63)
